### ISSUE TITLE
[enhance feature]: allow extensive customize strings in android

### ISSUE THREAD

**Issue by hieplam** [zoho/SalesIQ-Mobilisten-ReactNative#22]
Hi, localization in android work fine, however there are some text that can not change like the images below

Can we expose that to `strings.xml`. I searched all the text in locliaztion file but none of them relate to the moment time

<img width="431" alt="Screen Shot 2021-07-29 at 12 02 16" src="https://user-images.githubusercontent.com/10095360/127441271-b516381a-34d1-47e1-8401-53d03d823c89.png">
<img width="296" alt="Screen Shot 2021-07-29 at 12 01 31" src="https://user-images.githubusercontent.com/10095360/127441273-ac31f621-58ef-4e45-9c40-fee9442b8731.png">

**Comment by Angu-G** [comment]
Hi @hieplam, Message status indicating texts (like **Sent, Sending**) will be added for localization and customization in our next immediate update. Will look into the possibilities for the timestamp and let you know. Thank you!

**Comment by hieplam** [comment]
many thanks, i'm looking for your respone soon :)

**Comment by Angu-G** [comment]
Hello @hieplam, We have released an update(**v4.1.0**) with the customization for the Message status indicating texts (like **Sent, Sending**) and regarding the timestamp request, this is currently not possible since we are using some dynamic flows and we will try to include this too in our future updates. 
You can make use of the following attributes and let us know. Thank you!!
```
<string name="livechat.message.status.sent">Sent</string>
<string name="livechat.message.status.sending">Sending</string>
<string name="livechat.message.status.resend">Resend</string>
```

**Comment by Angu-G** [comment]
Hi @hieplam, Hope the customization helps. We are closing this issue now. You can write back us anytime. Thank you.
